lib/Transform/operators_supported.json

```diff
@@ -27,6 +27,22 @@
 		["Storage.Bytes", "Total storage required for MaxPool2D engine", "long"]
 	    ]		
 	},
+	{
+	    "name" : "torch.aten.mean.dim",
```

Comment:
Unrelated to the review, just to get it out there: I was thinking whether we might be able to generate this information using existing TableGen definitions. Let's discuss this in a separate context if you feel like this might be worthwhile.

Reply:
I like the idea. Sadly `torch-mlir` may be just about the worst dialect for this. Its operations are generated using `third-party/torch-mlir/python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/torch_ods_gen.py` which doesn't provide any descriptive information about the operands other than the type. The operation origins are in the `pytorch` op registry which may have more information, but I confess I only know of the connection and don't know the details.